source/_components/fritz.markdown

```diff
@@ -14,9 +14,38 @@ The `fritz` platform offers presence detection by looking at connected devices t
 
 ## Setup
 
-<div class='note warning'>
-If not running Hass.io it might be necessary to install additional packages: <code>sudo apt-get install python3-lxml libxslt-dev libxml2-dev zlib1g-dev</code>
-If you installed Home Assistant in a virtualenv, run the following commands inside it: <code>pip3 install lxml</code>; be patient this will take a while.</div>
+### Hass.io
+
+No additional steps are required.
+
+### Docker
+
+No additional steps are required.
+
+### Hassbian
+
+You need to install four packages, and then install a Python library in the virtual environment:
+
+```bash
+$ sudo apt-get install python3-lxml libxslt-dev libxml2-dev zlib1g-dev
```

Comment:
We should remove `$`, since there is no output displayed. It violates Markdownlinting rules MD014 and makes it hard for the user to copy and paste.

More information and reasoning: <https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md014---dollar-signs-used-before-commands-without-showing-output>

Reply:
Yet, everywhere else, we tell people to put the $ in...

Indeed, I'm pretty sure you've rejected PRs that _don't_ do that

Reply:
I've rejected PR's that DID do this and been actively removing it.

It is also documented in our own documentation right here:

> When you're writing code that is to be executed on the terminal, do not prefix them with $, since this makes it hard to copy and paste the commands. However, an exception is made when there is a need to distinguish between typed commands and command output. In those cases, prefixing the commands with a $ is required.

https://developers.home-assistant.io/docs/en/documentation_create_page.html#embedding-code

Reply:
Well, that's relatively new...

Reply:
Ah yes, you changed it last month...

Reply:
We (me & @klaasnicolaas) have been applying it way longer. I noticed it was not in the dev docs when updating it for some other larger changes made (removal of `linkable_title`).

Reply:
Yeah, unfortunately the changes haven't been communicated, beyond that update, so lots of folks aren't going to know...

Reply:
Guys come on please stick to the point (this PR), thank you!